src/viam/app/app_client.py

```diff
@@ -1170,7 +1170,8 @@ async def upload_module_file(self, module_file_info: Optional[ModuleFileInfo], f
             await stream.send_message(request_module_file_info)
             await stream.send_message(request_file, end=True)
             response = await stream.recv_message()
-            assert response is not None
+            if not response:
+                await stream.recv_message()  # causes us to throw appropriate gRPC error.
```

Comment:
You do `recv_trailing_metadata` in the other two functions, but `recv_message` here. Any difference?

Reply:
Oh, thanks for catching that! Mistake on my part, it should be `recv_trailing_metadata` :) Fixed!